cmds/core/backoff/backoff.go

```diff
@@ -31,16 +31,40 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"log"
 	"os"
+	"os/exec"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
```

Comment:
While you are on it, maybe it makes sense to bump the dependency to the newest version `v5`?